tools/generate-bidi-types.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+outfile=src/protocol/webdriver-bidi.ts
+
+cat <<EOF > $outfile
+/**
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * THIS FILE IS AUTOGENERATED. Run \`npm run bidi-types\` to regenerate.
+ * @see https://github.com/w3c/webdriver-bidi/blob/master/index.bs
+ */
+
+EOF
+
+cddlconv "$1" >> $outfile
+
+# Format twice because the first format doesn't get everything.
```

Comment:
Should we call `npm run format` instead, to decrease coupling between tools?

Reply:
Also note, there used to be a `npm run prettier` that was removed. It's fine to reintroduce it in case you dislike `npm run format` (which invokes all formatters).

Reply:
Using `npm run format` is overkill since it runs all the precommits. We only need prettier. I think it's safe to leave it like this; we don't want to expose prettier in `package.json` because that may encourage use of it.

Reply:
Here's the problem: The way it is currently, it forces the user to have `prettier` locally installed and in `$PATH`.

If you do it via `npm`, then there is no need to have it locally installed, nor in the `$PATH`. `npm` properly includes `node_modules/.bin`.

Reply:
Hmm, then let me use `npx`

Reply:
That's also fine. And it will guarantee we use the same version as in `package.json`.